cpp-client/deephaven/dhcore/include/public/deephaven/dhcore/column/array_column_source.h

```diff
@@ -196,51 +201,58 @@ class GenericArrayColumnSource final : public deephaven::dhcore::column::Mutable
   using RowSequence = deephaven::dhcore::container::RowSequence;
 
 public:
-  static std::shared_ptr<GenericArrayColumnSource> Create() {
-    auto elements = std::make_unique<T[]>(0);
-    auto nulls = std::make_unique<T[]>(0);
-    return CreateFromArrays(std::move(elements), std::move(nulls), 0);
-  }
-
-  static std::shared_ptr<GenericArrayColumnSource> CreateFromArrays(std::unique_ptr<T[]> elements,
-      std::unique_ptr<bool[]> nulls, size_t size) {
-    return std::make_shared<GenericArrayColumnSource>(Private(), std::move(elements), std::move(nulls),
-        size);
+//  static std::shared_ptr<GenericArrayColumnSource> Create() {
```

Comment:
Commented out code?

Reply:
At this point I think it can just be deleted.